.env.development

```diff
@@ -1,4 +1,4 @@
-FLASK_APP=app.py
```

Comment:
Shall we update the Readme files with the new application setup changes if required

Reply:
I don't think it is required.